src/Sarif/Writers/PrereleaseCompatibilityTransformer.cs

```diff
@@ -87,6 +87,8 @@ public static class PrereleaseCompatibilityTransformer
                         }
                         case "http://json.schemastore.org/sarif-2.1.0-rtm.5":
                         case "https://schemastore.azurewebsites.net/schemas/json/sarif-2.1.0-rtm.5.json":
+                        case "http://json.schemastore.org/sarif-2.1.0-rtm.6":
+                        case "https://schemastore.azurewebsites.net/schemas/json/sarif-2.1.0-rtm.6.json":
```

Comment:
Here we keep the rtm.5 and add rtm.6 because both version no need to apply change.